src/client/client.rs

```diff
@@ -521,7 +522,7 @@ where
     }
 }
 
-impl<C, B> tower_service::Service<Request<B>> for Client<C, B>
+impl<C, B> Service<Request<B>> for Client<C, B>
```

Comment:
Ah, yea, we probably don't need the `Service` impls in the client. Most of these parts are being moved to `hyper-util`, which _can_ integrate with `tower::Service`, since clients are a place I think backpressure makes more sense.